.github/workflows/vast.yaml

```diff
@@ -953,14 +898,45 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Run Integration Tests
         run: |
-          if [[ "$GITHUB_EVENT_NAME" == "pull_request" ]]; then
-            export VAST_CONTAINER_REF="${{ needs.determine-version.outputs.head-ref-slug }}"
-          else
-            export VAST_CONTAINER_REF="${{ github.sha }}"
-          fi
+          export VAST_CONTAINER_REF="${{ needs.determine-version.outputs.vast-container-ref }}"
           export VAST_CONTAINER_REGISTRY=ghcr.io
           docker/test
 
+
+  example-notebooks:
+    needs:
+      - build-docker
+      - determine-version
+    if: github.event_name != 'workflow_dispatch'
+    name: Build example notebooks
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          submodules: recursive
```

Comment:
I don't think we need either of these here.

Reply:
👍